test/acceptance/mixins/results.js

```diff
@@ -4,4 +4,10 @@ module.exports = (target) => {
   target.shouldBeOnResultsPage = function() {
     this.getText('h1').should.contain('records found for');
   };
+
+  target.clickFirstRecord = function() {
+    // browser.element gets just the first matching elements (browser.elements gets all matching ones)
+    const linkToFirstRecordDetails = browser.element('#records a').value.ELEMENT;
+    browser.elementIdClick(linkToFirstRecordDetails);
```

Comment:
this.click('#records a') does work but is a bit dodgy, as "#records a" returns an array, it happens to click the right one but I would prefer to explicitly click the first one